### PR TITLE
get user handler

### PR DESCRIPTION
**Pull request by rfpoulos**
@rcackerley @smith-taher I’m having a hell of a time getting the psql database to run locally, so I haven’t really been able to test anything.  I changed a couple of value requirements in the user database (serial id, requireing a username, password, email, ect.) . If possible, pls try and clone the code and run locally.  I'll work with a TA to get the database to start working on my machine sometime today.

### CODE REVIEW

**Comment by rfpoulos** [comment]
I should also mention that I named the sql queries functions to have 'db' in the name to keep them straight with the handlers.  I also just realized that my handler isn't sending back any info yet . . .so a promise is coming soon.

**Comment by rfpoulos** [comment]
Added the promise, but I still don't know if it works because haven't been able to get the database to run.   To clarify, I don't think it is a problem with the database, but that I'm a little behind on my psql knowledge/commands.

File: server.js [hunk 1]
```diff
@@ -5,37 +5,65 @@ const getNews = require('./newsAPI');
...
+  response.end('404, nothing here!');
+};
+
 let routes = [
   { method: 'DELETE', path: /^\/users\/([0-9]+)$/, handler: deleteUser },
   { method: 'GET', path: /^\/users\/([0-9]+)$/, handler: getUser },
   { method: 'PUT', path: /^\/users\/([0-9]+)$/, handler: putUser },
   { method: 'GET', path: /^\/users\/?$/, handler: getUsers },
   { method: 'POST', path: /^\/users\/?$/, handler: postUser },
+  { method: 'GET', path: /^\/signin\/?$/, handler: signIn },
```
Comment (rcackerley) [inline server.js]: I realized last night that a 'GET' method cannot have a body attached to it so can you change it to a 'POST' method. Other than that - things look good. Once you merge this - I'll merge my code that validates a userID and password against the DB.

Comment (rfpoulos) [reply server.js]: updated.  were you able to use the method with your local db?

Comment (rcackerley) [reply server.js]: yeah, I wrote a test one last night and passed some test data to it and then wrote a validation function to call the db and compare the two.

**Pull request merged**